tests/common/dualtor/control_plane_utils.py

```diff
@@ -0,0 +1,115 @@
+"""Contains functions used to verify control plane(APP_DB, STATE_DB) values."""
+
+APP_DB = 0
+STATE_DB = 6
+APP_DB_MUX_STATE_FIELDS = {
+    "MUX_CABLE_TABLE": "state",
+    "HW_MUX_CABLE_TABLE": "state",
+    "MUX_CABLE_RESPONSE_TABLE": "response"
+}
+STATE_DB_MUX_STATE_FIELDS = {
+    "MUX_CABLE_TABLE": "state",
+    "HW_MUX_CABLE_TABLE": "state"
+}
+
+
+def _keys(duthost, db, key_pattern):
+    """Run Redis command keys over db on duthost."""
+    command = "redis-cli --raw -n {db} keys '{key_pattern}'".format(db=db, key_pattern=key_pattern)
+    keys_result = duthost.shell(command)
+    if not keys_result["stdout"].strip():
+        raise ValueError("No keys match key pattern {}".format(key_pattern))
+    return [line.strip() for line in keys_result["stdout_lines"]]
+
+
+def _hgetall(duthost, db, key):
+    """Run Redis command hgetall over db on duthost."""
+    command = "redis-cli --raw -n {db} hgetall '{key}'".format(db=db, key=key)
+    lines = duthost.shell(command)["stdout_lines"]
+    return {lines[i]: lines[i + 1] for i in range(0, len(lines), 2)}
+
+
+def expect_app_db_values(duthost, intf_name, state):
```

Comment:
Can you also include utils to check state for all intf instead of having the caller call this method multiple times? For eg, a test where all intfs on one TOR are active/standby

The same comment applies for state db as well

Reply:
Thanks, @neethajohn, updated.